README.md

```
@@ -18,6 +18,18 @@ dependencies:
   flutter_pdfview: ^1.4.5
 ```
 
+#### Trying the 1.5.0 beta
+
+`1.5.0-beta.1` ports the native implementations to Kotlin (Android) and Swift (iOS) with no
+public Dart API changes. Pre-releases are not picked up by a `^` constraint, so pin it explicitly:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)